packages/glsp-playwright/package.json

```diff
@@ -45,15 +45,13 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
+    "@playwright/test": "^1.40.1",
+    "@theia/playwright": "^1.45.0",
     "@types/uuid": "^9.0.0",
     "@vscode/test-electron": "^2.3.2",
     "concurrently": "^7.6.0",
     "tsc-alias": "^1.8.2"
   },
-  "peerDependencies": {
-    "@playwright/test": "^1.32.1",
-    "@theia/playwright": "^1.39.0"
```

Comment:
Please also keep `@playwright/test` and `@theia/playwright` as peerDependencies.
They need to be present when the package is consumed from npm. So we should explicitly delcare them as `peerDependencies`.